routers/web/web.go

```diff
@@ -297,16 +205,16 @@ func RegisterRoutes(m *web.Route) {
 	// Especially some AJAX requests, we can reduce middleware number to improve performance.
 	// Routers.
 	// for health check
-	m.Get("/", routers.Home)
+	m.Get("/", Home)
 	m.Get("/.well-known/openid-configuration", user.OIDCWellKnown)
 	m.Group("/explore", func() {
 		m.Get("", func(ctx *context.Context) {
 			ctx.Redirect(setting.AppSubURL + "/explore/repos")
 		})
-		m.Get("/repos", routers.ExploreRepos)
-		m.Get("/users", routers.ExploreUsers)
-		m.Get("/organizations", routers.ExploreOrganizations)
-		m.Get("/code", routers.ExploreCode)
+		m.Get("/repos", explore.Repos)
+		m.Get("/users", explore.Users)
+		m.Get("/organizations", explore.Organizations)
+		m.Get("/code", explore.Codes)
 	}, ignExploreSignIn)
```

Comment:
Hmm could we offload this to the explore package? Maybe for a different PR I guess...

Reply:
yes would say so